### DISCUSSION TITLE
can i add PDFs to a page - is this possible?

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#33718] [Other Feature Feedback, Questions, & Ideas]
can i add PDFs to a page - is this possible?

am i able to add some pdf documents to a page - eg here . 

https://github.com/fsj-digital/kurs

how to do this - how to place them on the page!? 

**update**: the solution is this one: 

 to embed the PDF in the web page.  we can upload the CV PDF file to the repository, then use some HTML to embed it.
`<object data="myfile.pdf" width="1000" height="1000" type='application/pdf'/>`

i am trying to test several modes of embedding: which one runs with some effect - which one does work. no one so far: What to do now!?

cf.: https://github.com/mmistakes/minimal-mistakes/issues/2361

which one runs with some effect - which one does work. no one so far: What to do now!? 

cf.: https://github.com/mmistakes/minimal-mistakes/issues/2361

neither this one 
--- #
title: "Résumé"
permalink: /resume/
header:
---
[See it here :)](http://ToadHanks.github.io/images/mihir_resume_2019v5.pdf)
Then I get a link that opens to new tab instead of a pdf embedded page. However, I have saved to pdf to image, and was able to display that in the page.

nor this one: 

#https://raw.githubusercontent.com/belovanna/belovanna.github.io/b64e6c285e0e057f9bc9a1723a4eb767446a1947/_pages/curriculum.md
Effect: https://belovanna.github.io/Curriculum/

what can i do  now!?

**Comment by UjwalKandi** [comment]
Hey @fsj-digital,

Well, I saved the pdf file in the same repo of my site and used its path to retrieve it from embedded [word](https://github.com/).

https://[repo].github.io/assets/my_file.pdf

**Reply by fsj-digital** [reply]
hello and good day - many many thanks - i will try this out. 
im glad to be part of this forum 
have a great day

**Reply by fsj-digital** [reply]
Hello dear friends, thanks for all and of course for the explanation... 

well i do not want to load it down i want to open it in a new window <a href="username.github.io/folder/document.pdf" target="_blank">PDF.</a> how to do this?: <a href="[publications.jrc.ec.europa.eu/repository/bitstream/JRC117910/…]

(https://publications.jrc.ec.europa.eu/repository/bitstream/JRC117910/jrc117910_dihs_survey_jrc_report_pubsy_online.pdf)" target="_blank">PDF-Text.</a>
